docs/API_ref.md

```diff
@@ -63,5 +64,15 @@
 | tolerations         | [[]corev1. Toleration](https://pkg.go.dev/k8s.io/api/core/v1#Toleration)                  | Tolerations defines the list of tolerations to be applied to Velero Deployment/Restic daemonset                                                             |
 | resourceAllocations | [corev1.ResourceRequirements](https://pkg.go.dev/k8s.io/api/core/v1#ResourceRequirements) | Set specific resource  `limits`  and  `requests`  for the Velero/Restic pods. For more information, go  [here] ( config/resource_req_limits.md). |
 
+### Features
+| Property  | Type      | Description                                                                                                                               |
+|-----------|-----------|-------------------------------------------------------------------------------------------------------------------------------------------|
+| DataMover | DataMover | DataMover defines the various config for DPA data mover                                                             |
```

Comment:
Did you forget your own commits? :D
https://github.com/openshift/oadp-operator/blob/a02487302a02ce9cb014c17a8fe1072c7ae4a805/bundle/manifests/oadp.openshift.io_dataprotectionapplications.yaml#L446-L450